.github/workflows/test.yml

```diff
@@ -0,0 +1,73 @@
+name: vcr ci
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
```

Comment:
I'm curious if this alias for default branch does the right thing. It's be cool.